script/zonemaster_backend_rpcapi.psgi

```diff
@@ -103,6 +107,34 @@ my $router = router {
 	};
 };
 
+# Returns a Log::Any-compatible log level string, or throws an exception.
+sub get_loglevel {
+    my $value = $ENV{ZM_BACKEND_RPCAPI_LOGLEVEL} || 'warning';
+    for my $method ( logging_methods(), logging_aliases() ) {
```

Comment:
There is already another environment variable used as an option to this script, "ZONEMASTER_BACKEND_CONFIG_FILE". I think it would make sense if both om them come from the same name space, i.e. both being "ZONEMASTER_BACKEND_*" or both being "ZM_BACKEND_*".